onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -198,10 +198,11 @@ def aten_binary_cross_entropy_backward(
     raise NotImplementedError()
 
 
-def aten_celu(self, alpha: float = 1.0):
+@torch_op("aten::celu")
+def aten_celu(self: TFloat, alpha: float = 1.0) -> TFloat:
```

Comment:
```suggestion
def aten_celu(self: FLOAT, alpha: float = 1.0) -> FLOAT:
```

because only one type is supported. TFloat can be one of a few types.